crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -576,13 +824,19 @@ impl HolochainP2pHandler for HolochainP2pActor {
     #[tracing::instrument(skip(self), level = "trace")]
     fn handle_publish(
         &mut self,
-        dna_hash: DnaHash,
-        from_agent: AgentPubKey,
-        request_validation_receipt: bool,
-        dht_hash: holo_hash::AnyDhtHash,
-        ops: Vec<(holo_hash::DhtOpHash, holochain_types::dht_op::DhtOp)>,
-        timeout_ms: Option<u64>,
+        _dna_hash: DnaHash,
+        _from_agent: AgentPubKey,
+        _request_validation_receipt: bool,
+        _dht_hash: holo_hash::AnyDhtHash,
+        _ops: Vec<(holo_hash::DhtOpHash, holochain_types::dht_op::DhtOp)>,
+        _timeout_ms: Option<u64>,
     ) -> HolochainP2pHandlerResult<()> {
+        // TODO - FIXME - disabling publish right now
+        //                it's not designed well
+        //                and we can rely on full-sync gossip
+        tracing::debug!("IGNORING PUBLISH - TRUSTING IN GOSSIP");
```

Comment:
So ops need be self gossiped so we know we are the authority. By disabling publish you will never be able to gossip anything